WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Method/SimplePaymentsMethodsViewModel.swift

```diff
@@ -221,12 +237,12 @@ private extension SimplePaymentsMethodsViewModel {
     /// Observes the store CPP state and update publish variables accordingly.
     ///
     func bindStoreCPPState() {
-        cppStoreStateObserver
-            .statePublisher
-            .map { $0.isCompleted }
-            .removeDuplicates()
-            .assign(to: &$showPayWithCardRow)
-        cppStoreStateObserver.refresh()
+//        cppStoreStateObserver
```

Comment:
If this is not needed, should we remove the method altogether?

Reply:
This was removed in 5ec5aa16e1dc6ca64d8376ad29ab85d8042482ca, the commit that helps with testing by making the `Card` option always show. Normally, it only shows if you've got `.complete` as your onboarding state. I'll revert that before I merge.

Reverting that will mean it's impossible to get the onboarding flow on Simple Payments. I'm going to log further tickets for the next iteration to improve on this, pending input from @joe-keenan on the flows as well 😊